grants/search.js

```diff
@@ -421,12 +414,28 @@ async function buildMatchCriteria(queryParams) {
         match.$and.push({ 'beneficiaryLocation.country': queryParams.country });
     }
 
+    if (match.$and.length === 0) delete match.$and;
```

Comment:
Could you wrap this in braces, even though it's a one-liner? I get a bit edgy with that syntax...